conda_smithy/feedstock_content/ci_support/disabled.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="92" height="20">
```

Comment:
Nope. It is not copied thanks to this [line]( https://github.com/conda-forge/conda-smithy/pull/228/files#diff-3a2c4a048580db27ad9db93e2780b511R304 ).